docs/doxygen/pages.dox

```diff
@@ -0,0 +1,53 @@
+/**
+@mainpage Overview
+@anchor jobs
+@brief AWS IoT Jobs client library, part of the AWS IoT Device SDK for Embedded C
+
+<p>
+AWS IoT jobs can be used to define a set of remote operations that are sent to and executed on one or more devices connected to AWS IoT.
+For documentation of the service, please see the [AWS Iot Developer Guide](https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html).
+Interactions with the jobs service use MQTT, a lightweight publish-subscribe protocol.
+This library provides a convenience API to compose and recognize the MQTT topic strings used by the jobs service.
+The library is written in C and designed to be compliant with ISO C90 and MISRA C.
+It has no heap allocation and proofs showing safe memory use, making it suitable for IoT microcontrollers, but also fully portable to other platforms.
```

Comment:
```suggestion
It has no heap allocation, and contains proofs showing safe memory use, thereby, making it suitable for IoT microcontrollers, but also fully portable to other platforms.
```

Reply:
Pass on this...  I'll change the wording some other way.